### PR TITLE
docs: add clip guide

### PR DESCRIPTION
**Pull request by jupyterjazz**
Example on fine-tuning `CLIP` for text to image retrieval task.

[Link](https://ft-docs-resnet-guide--jina-docs.netlify.app/docs-resnet-guide/3_finetuner_in_action/3_3_text_to_image_using_clip/)

### CODE REVIEW

File: docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md [hunk 1]
```diff
@@ -1,3 +1,110 @@
...
+
+## Preparing data
+Training and evaluation data are already prepared and pushed to Hubble following the [instructions](../2_step_by_step/2_4_create_training_data.md).
+You can either pull the data:
+```python
+from docarray import DocumentArray
+train_data = DocumentArray.pull('clip-fashion-train-data')
+eval_data = DocumentArray.pull('clip-fashion-eval-data')
+```
+Or specify given `DocumentArray` names (`clip-fashion-train-data` and `clip-fashion-eval-data`) directly to the finetuner.
```
Comment (gmastrapas) [inline docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md:19]: directly to finetuner

File: docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md [hunk 2]
```diff
@@ -1,3 +1,110 @@
...
+```python
+run.save_model('clip-model')
+```
+
+## Evaluation and performance
+Currently, we don't have a user-friendly way to get evaluation metrics from the `EvaluationCallback` we initialized previously.
+What you can do for now is to call `run.logs()` in the end of the run and see evaluation results:
+
+```bash
+[10:37:49] DEBUG    Metric: 'model_average_precision' Value: 0.30105                                     __main__.py:217
```
[the remark this answers is not in the document]
Comment (jupyterjazz) [reply docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md:95]: No, it's just a way to show how to see metrics from logs. I couldn't run these today, runs were stuck and now it would take too much time.

File: docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md [hunk 3]
```diff
@@ -1,3 +1,107 @@
...
+        model='openai/clip-vit-base-patch32',
+        run_name='clip-fashion',
+        train_data='clip-fashion-train-data',
+        eval_data='clip-fashion-eval-data',
+        epochs=5,
+        learning_rate= 1e-5,
+        callbacks=[BestModelCheckpoint(), EvaluationCallback(query_data='clip-fashion-eval-data')],
+        loss='CLIPLoss',
+        image_modality='image',
+        text_modality='text',
```
Comment (bwanglzu) [inline docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md:47]: can we add some sentence below to explain the relationship between these two arguments and how we create DocumentArray?

**Review by bwanglzu — COMMENTED** [review]
left some comment

File: docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md [hunk 4]
```diff
@@ -1,3 +1,107 @@
 # Text to image search using CLIP
 
-Coming soon!
\ No newline at end of file
+This guide will showcase fine-tuning a `CLIP` model for text to image retrieval.
+
+## Task overview
+We'll be fine-tuning CLIP on a fashion captioning dataset which contains information about fashion products.
+
+For each product the dataset contains a title and images of multiple variants of the product. We constructed a parent `Document` for each picture, which contains two chunks: an image document and a text document holding the description of the product.
```
Comment (jemmyshin) [inline docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md:8]: if a product has multiple images, will the image document be: `Document(chunks=[Document(image_0), Document(image_1])`?

Comment (jupyterjazz) [reply docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md:8]: Document(chunks=[Document(image_0), Document(title)], Document(chunks=[Document(image_1), Document(title)], etc.
